home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -721,6 +727,339 @@ paths:
         default:
           $ref: '#/components/responses/InternalServerError'
 
+  /houses/{house_id}/apartments:
+    get:
+      tags:
+        - apartment
+      summary: Get all apartments
+      description: The endpoint gets all apartments of the house
+      operationId: queryApartment
+      parameters:
+        - $ref: 'p_house_id'
+        - $ref: '#/components/parameters/page_number'
+        - $ref: '#/components/parameters/page_size'
+        - $ref: '#/components/parameters/sort'
+        - $ref: '#/components/parameters/filter'
+        - $ref: '#/components/parameters/id'
+        - $ref: '#/components/parameters/apartment_number'
+        - $ref: '#/components/parameters/apartment_area'
+      responses:
+        200:
+          $ref: '#/components/responses/ApartmentsResponse'
+        '400':
```

Comment:
It would be nice to remove the quotes on the response codes.

Reply:
changed, thanks